imap_processing/lo/l0/utils/bit_decompression.py

```diff
@@ -0,0 +1,59 @@
+"""Decompression for 8 to 12, 8 to 16, and 12 to 16 bits."""
+from enum import Enum
+from pathlib import Path
+
+import numpy as np
+
+
+class Decompress(Enum):
+    """Decompression options."""
+
+    DECOMPRESS8TO12 = "8_to_12"
+    DECOMPRESS8TO16 = "8_to_16"
+    DECOMPRESS12TO16 = "12_to_16"
+
+
+def decompression_tables():
+    """Load all decompression tables into a dictionary."""
+    # load all decompression tables
+    return {
+        enum: np.loadtxt(
+            Path(__file__).parent.parent / f"decompression_tables/{enum.value}_bit.csv",
+            delimiter=",",
+            skiprows=1,
+        )
+        for enum in Decompress
+    }
```

Comment:
I _think_ you can make this an attribute of the module and then not have to re-load them on every packet even, just once on module import.

```suggestion
DECOMPRESSION_TABLES = {
        enum: np.loadtxt(
            Path(__file__).parent.parent / f"decompression_tables/{enum.value}_bit.csv",
            delimiter=",",
            skiprows=1,
        )
        for enum in Decompress
    }
"""Decompression tables stored as a lookup dictionary."""
```